slides/intro.md

```diff
@@ -0,0 +1,157 @@
+---
+theme: csc-2019
+lang: en
+pagetitle: Introduction
+---
+
+# Kielipankki & CSC {.title}
+
+Corpora, language technology, computational linguistics, corpus linguistics, supercomputers and all that
+
+<div class="column">
+![](https://mirrors.creativecommons.org/presskit/buttons/88x31/png/by-sa.png)
+</div>
+<div class="column">
+<!-- <small> -->
+<!-- All materials (c) 2023 by CSC – IT Center for Science Ltd. -->
+<!-- This work is licensed under a **Creative Commons Attribution-ShareAlike** 4.0 -->
+<!-- Unported License, [http://creativecommons.org/licenses/by-sa/4.0/](http://creativecommons.org/licenses/by-sa/4.0/) -->
+<!-- </small> -->
+</div>
+
+# Kielipankki <img src="img/kielipankki_logo.png" style="display:inline-block;position:absolute;top:-20px;">
+
+## AKA The Language Bank of Finland
+
+* We create, solicit and receive corpora
+
+* We enrich the corpora with NLP tools
+
+* We make available our enriching tools, plus others
+
+* Hundreds of corpora, tens of billions of words
+
+# CSC
+
+* Owns and operates computational resources and services
+
+* Our own cloud, network, supercomputers, storage solutions
+
+* Kielipankki uses CSC resources, and helps language-oriented users use them
+
+# What can we do with language?
+
+## Computational linguistics
+<img src="img/shift_reduce.gif"><img src="img/lexc.png"><img src="img/praat_sound_file.png" style="width:25%">
+
+<!-- Image credits? http://www.ling.helsinki.fi/~klinden/pubs/sfcm09hfst.pdf https://www.sciencedirect.com/science/article/abs/pii/S000437021830540X https://joeystanley.com/blog/a-tutorial-on-extracting-formants-in-praat -->
+
+
+* Linguistics from a computer science perspective: formalisms, parsing, rule-writing
+
+* We have and use tools for this
+
+* We will not focus on this in this course
+
+# What can we do with language?
+
+## Language technology
+
+* Same thing but in the 90's-2000's <div class="column"><img src="img/like_cluster_for_liikenne.png"></div>
+
+* More application-oriented and uses statistics
+
+* More computation
+
+* We have an bonus problem set on topic modeling in this course
+
+# What can we do with language?
+
+## Corpus linguistics
+
+* Statistics on a body of text <div class="column"><img src="img/lemmarank.png"></div>
+
+* Can use tool-generated information: morphology, syntax, semantics etc.
+
+* Arguably Kielipankki's main focus, we will do a project in this course
+
+# What can we do with language?
+
+## Modern large-scale machine learning
+
+* This is what you read about in the headlines <div class="column" style="width:30%"><img src="img/shoggoth.jpeg"></div>
+
+* Generic statistical approaches, no linguistic content
+
+* Huge amounts of data and computation, cutting-edge applications
+
+* A lot of this goes on at CSC, but we won't do it in this course
+
+# Kielipankki resources: Korp
+
+* A browser for corpora <div class="column" style="width:50%"><img src="img/korp_screenshot.png"></div>
```

Comment:
This slide could be a bit more balanced if it had two columns somewhat like the eldritch LLM suggestion has. If reading the text is not required, this is just an aesthetic bonus though, not a requirement.